packages/terra-collapsible-menu-view/tests/wdio/collapsible-menu-view-spec.js

```diff
@@ -45,7 +46,7 @@ Terra.describeViewports('Responsive Single Item', ['tiny', 'large'], () => {
   it('should display responsive single item', () => {
     browser.url('/raw/tests/cerner-terra-framework-docs/collapsible-menu-view/collapsible-menu-view-single-item');
     $('#root').moveTo({ xOffset: 0, yOffset: 0 });
-    Terra.validates.element('responsive single item', { selector: '#root' });
+    Terra.validates.element('responsive single item', { selector: '#root', rules: ignoredA11y });
```

Comment:
why are a11y checks disabled ?

Reply:
Collapsible-Menu-View-Item-Group adds up a [<div> wrapper ](https://github.com/cerner/terra-framework/blob/6f7008185f7510d55b9ead4ec6f9cde1078e72c4/packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx#L80) within terra-menu which is apparently `ul` element. `ul` should always have `li` as it's child element hence additional `li` element has been added.

Since `ul` has role `menu` we cannot have place `li` without role as it would expect children to have either role `menuItem`, `menuItemCheckbox` OR `menItemRadio`. Since it is group element above roles cannot be added to `li` wrapper element. this is the reason why we need ally rules disabled for collapsible-menu-view tests.